examples/istio/quick_start.yaml

```diff
@@ -364,6 +364,12 @@ webhooks:
     namespaceSelector:
       matchLabels:
         opa-istio-injection: enabled
+    objectSelector:
+      matchExpressions:
+      - key: sidecar.opa-istio.io/inject
+        operator: NotIn
+        values:
+        - "false"
     failurePolicy: Fail
```

Comment:
Let's revert the changes in the quick start as we've already documented this in the README.

Reply:
Oh, I'm sorry.
I fixed it.